config/locales/en/helpers.yml

```diff
@@ -25,6 +25,12 @@ en:
       adult_simple_caution: Simple caution
       adult_conditional_caution: Conditional caution
 
+    CAUTION_TYPE_HINTS: &CAUTION_TYPE_HINTS
```

Comment:
Maybe the dictionary can have just 2 keys, one for `simple_caution_hint` and another for `conditional_caution_hint` and then we use these values in the `caution_type_options`?
Just an idea. Otherwise a dictionary seems a bit like of no use.
Also we can delete the old hint texts as they are no longer used I believe.

Reply:
Ah yes, good shout, will do this!